test/integration/scheduler/queueing/queueinghint/queue_test.go

```diff
@@ -34,8 +36,12 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			continue
 		}
 		// Note: if EnableSchedulingQueueHint is nil, we assume the test should be run both with/without the feature gate.
+        var prefix string
+        if tt.EnableSchedulingQueueHint != nil {
```

Comment:
Why only if it's non nil?

Reply:
Ops, I was mistaken. I'd like to `tt.EnablePlugins != nil`